Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -117,6 +117,8 @@ defineSuite([
     var tilesetReplacementWithViewerRequestVolumeUrl = 'Data/Cesium3DTiles/Tilesets/TilesetReplacementWithViewerRequestVolume/tileset.json';
 
     var tilesetWithExternalResourcesUrl = 'Data/Cesium3DTiles/Tilesets/TilesetWithExternalResources/tileset.json';
+    var tilesetUrlWithContentUri = 'Data/Cesium3DTiles/Tilesets/TilesetWithContentUri/tileset.json';
```

Comment:
Will there be a pull request into https://github.com/AnalyticalGraphicsInc/3d-tiles-tools for this new tileset?

Reply:
Created PR https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/pull/149 for running changes. I would think review that last after the Cesium changes.